cmd/account-assertions/main.go

```diff
@@ -23,22 +23,23 @@ import (
 	"log"
 
 	"github.com/CanonicalLtd/serial-vault/account"
-	"github.com/CanonicalLtd/serial-vault/service"
+	"github.com/CanonicalLtd/serial-vault/config"
+	"github.com/CanonicalLtd/serial-vault/datastore"
 )
 
 func main() {
```

Comment:
It just started there and has never been moved. I've been wary of messing up the deployed snap and the docker script has the location coded there. This may be a good time to move it, but it will result in more changes i.e. to the snap files... brace yourself!